### PR TITLE
Add form for blobstore

### PR DESCRIPTION
**Pull request by amuamushu**
A file input option is added to the form. Also, the form's POST request is updated so that content is passed into Blobstore instead of directly to /data. 

Image upload form appearing on the page: https://screenshot.googleplex.com/K8RZUFTpgqN.png
Blobstore file Url printed to the console: https://screenshot.googleplex.com/F7EDeGFBXJ9.png

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -113,6 +113,7 @@ private String convertToJsonUsingGson(List toConvert) {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("doPost()");
```
Comment (billie-cat) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:116]: Why is this needed?

Comment (amuamushu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I think I was planning on leaving it until I finished implemented all of blobstore to make sure it works, but I just removed it so it won't be merged to master.

Done!
